gammapy/image/tests/test_lists.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+from gammapy.datasets import FermiGalacticCenter
+import numpy as np
+
+
+def test_to_cube():
+    sky_cube_original = FermiGalacticCenter.diffuse_model()
+    image_list = sky_cube_original.to_image_list()
+    assert np.array_equal(image_list.skymaps[0].data, sky_cube_original.data[0])
```

Comment:
Also: please move this assert down a little.

Tests are easier to read if they follow the pattern:
```
# setup
# execution
# asserts
```
instead of mixing execution and asserts.